.github/workflows/rust.yml

```diff
@@ -220,6 +220,8 @@ jobs:
         run: cargo check --profile ci --no-default-features -p datafusion --features=unicode_expressions
       - name: Check parquet encryption (parquet_encryption)
         run: cargo check --profile ci --no-default-features -p datafusion --features=parquet_encryption
+      - name: Check parquet encryption (parquet, parquet_encryption)
+        run: cargo check --profile ci --no-default-features -p datafusion --features=parquet,parquet_encryption
```

Comment:
This seems like it should be redundant but then I noticed that the `parquet_encryption` feature doesn't actually depend on the `parquet` feature. Maybe we should remove this extra check run as the one above already tests the `parquet_encryption` feature, and update `datafusion/core/Cargo.toml` so that the `parquet_encryption` feature depends on the `parquet` feature rather than just `dep:parquet`?